supervisor/jobs_api.go

```diff
@@ -142,13 +142,23 @@ func (self JobAPI) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 		re := regexp.MustCompile(`^/v1/job/([a-fA-F0-9-]+)/run`)
 		id := uuid.Parse(re.FindStringSubmatch(req.URL.Path)[1])
+		taskChan := make(chan string)
 
 		self.Tasks <- &db.Task{
-			Op:      db.BackupOperation,
-			Owner:   params.Owner,
-			JobUUID: id,
+			Op:           db.BackupOperation,
+			Owner:        params.Owner,
+			JobUUID:      id,
+			TaskUUIDChan: taskChan,
+		}
+
+		var taskUUID string = <-taskChan
+		if taskUUID == "" {
+			w.WriteHeader(500)
+			JSONLiteral(w, fmt.Sprintf(`{"ok":"error"}`))
+		} else {
+			w.WriteHeader(200)
+			JSONLiteral(w, fmt.Sprintf(`{"ok":"scheduled","task_uuid":"%s"}`, taskUUID))
```

Comment:
as above, check on usage of `JSONLiteral` elsewhere and either fix or ticket.  I want to avoid use of JSONLiteral that could run afoul of quoting antics (this one doesn't but sets precedence for fmt.Sprintf-driven "literals")